pkg/storaged/dialog.jsx

```diff
@@ -1245,6 +1245,8 @@ export const TeardownMessage = (usage, expect_single_unmount) => {
         if (use.block) {
             const name = teardown_block_name(use);
             let location = use.location;
+            if (location && location.startsWith("/var/lib/cockpit/btrfs/"))
+                return;
```

Comment:
What about introducing a new mount option `x-cockpit-tmp-mount` and then filtering based on that mount option.

Reply:
Yeah, this is a bit random and also not totally trivial. We do want Cockpit to teardown this mount point because otherwise we couldn't format a block device with btrfs on it. The btrfs-tool can tolerate that. We just don't want to show it in the UI.

As to how to identify this mountpoint... I think just the path is enough and we don't need to identify more machinery. Also, we don't really have access to run-time mount options in Cockpit, only to the ones in fstab. Except for btrfs filesystems from the findmnt poll....

Reply:
Right, having mount options available... if that is btrfs specific then a general `x-cockpit-tmp-mount` which only works in btrfs case because of findmnt is not ideal. So we scratch the idea but might suggest adding a constant for it.

Reply:
> but might suggest adding a constant for it.

Yes, good idea.

Reply:
Done.